recipes/list_items/contact_info/contact_info.vue

```diff
@@ -7,12 +7,14 @@
     class="dt-contact-info"
   >
     <template #left>
-      <div
+      <button
         v-if="showAvatar"
-        class="d-ps-relative"
+        class="d-ps-relative d-bgc-transparent d-baw0 d-c-pointer"
         data-qa="contact-info-left"
+        :aria-label="avatarAriaLabel"
```

Comment:
Update it to `:aria-labelledby="avatarAriaLabel"` and a11y should be fixed.

Reply:
I applied the change as you suggested, but the a11y test was still failed: https://github.com/dialpad/dialtone-vue/actions/runs/6165158204/job/16732400646?pr=1164

do you have any idea about why?

Reply:
Oh right, there is another a11y failing test lol, lemme check

Reply:
ye seems like you cannot have a <button> inside a `dt-list-item` slot..

Reply:
We have sth similar in `feed_item_pill.vue` but we directly disabled a11y there.
Because the error was triggered on the `feed_item_pill_default.story.vue` not inside the component.

So maybe we can do the same here, but we definetly need to rewrite or rethink how we are using this `dt-item-layout`

So... you can disable the a11y in `recipes/list_items/contact_info/contact_info.stories.js`

```
a11y: {
      **disable: true,**
      config: {
        rules: [
          {
            id: 'aria-allowed-attr',
            enabled: false,
          },
          // Verify why contact info and avatar aren't getting unique IDs
          {
            id: 'duplicate-id',
            enabled: false,
          },
        ],
      },
    },
```

What you think @braddialpad, we could create a ticket to follow this issue and the one in `feed_item_pill_default.story.vue`. But for now disabling it here to not block this PR

Reply:
hmm weird that you cannot have a button inside an `li`.. didn't think this was invalid. But yeah we can investigate this in the other ticket